src/About/documentationSlice.js

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
This has no dependencies, maybe rather make it directly in typescript now, instead of having to rewrite later.